src/pages/waiting-room-to-car/components/bike-category-type/bike-category-type.ts

```diff
@@ -81,9 +87,24 @@ export class BikeCategoryTypeComponent implements OnChanges {
     this.testType  = (this.testType === BikeTestType.MOD1 || this.testType === BikeTestType.MOD2) ?
       this.testType : BikeTestType.MOD1;
     this.bikeCategoryDetails = this.bikeCategoryDetailProvider.getAllDetailsByTestType(this.testType);
+    this.initialCategory = this.testCategory;
+    this.testCategory = null;
```

Comment:
Is there any reason why this is null? Could it be also ` this. testCategory = this.testCategory;`  ?
and do we really need 2 fields for detecting a change in one of them? 
Do you update `this.initialCategory` anywhere else? will this solution deal with 2 changes in a row?

Reply:
If multiple changes occur in a row, then it would likely give inaccurate data. I'll look to change this.

Reply:
removed validation pending future task

Reply:
This has been changed since initial review